bazel/repository_locations.bzl

```diff
@@ -453,12 +453,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Nghttp2",
         project_desc = "Implementation of HTTP/2 and its header compression algorithm HPACK in Cimplementation of HTTP/2 and its header compression algorithm HPACK in C",
```

Comment:
Can we use this PR to fix this long outstanding typo?
`       project_desc = "Implementation of HTTP/2 and its header compression algorithm HPACK in C",`